schemas/questionnaire_v1.json

```
@@ -115,6 +115,29 @@
     "hub": {
       "$ref": "https://eq.ons.gov.uk/hub/definitions.json#/hub"
     },
+    "submission": {
+      "type": "object",
+      "properties": {
+        "button": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+          "description": "The text that appears on the submit button, when the questionnaire is complete."
+        },
+        "guidance": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+          "description": "The text guidance that appears, when the questionnaire is complete."
+        },
+        "title": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+          "description": "The title that will appear on the hub, when the questionnaire is complete."
```

This shouldn't mention hub, something like main heading or main page title should be used:
```suggestion
          "description": "The main heading when the questionnaire is complete."
```